src/components/Navbar/Navbar.tsx

```diff
@@ -205,14 +205,16 @@ export const Navbar: React.FC<NavbarProps> = ({
             >
               {topicMapSectionLabel}
             </Trigger>
-            <Trigger
-              className={styles.sectionTitle}
-              key={notesSectionLabel}
-              value={notesSectionLabel}
-              aria-label={notesSectionLabel}
-            >
-              {notesSectionLabel}
-            </Trigger>
+            {totalNotesToComplete !== 0 && (
```

Comment:
JavaScript's `number` type is verrrry brittle, so it's a little bit safer to use larger than, in case the number for some reason is 0.0000000003 one day.

```suggestion
            {totalNotesToComplete > 0 && (
```